src/containers/my-cooperations/accept-cooperation-close/AcceptCooperationClosing.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Typography } from '@mui/material'
+import { FC } from 'react'
+import AppButton from '~/components/app-button/AppButton'
+
+import { styles } from './AcceptCooperationClosing.styles'
+import { ErrorOutlineRounded } from '@mui/icons-material'
+import { SizeEnum } from '~/types'
+
+interface AcceptCooperationClosureProps {
+  user: string
+  onAccept: () => void
+}
+
+const AcceptCooperationClosing: FC<AcceptCooperationClosureProps> = ({
+  user,
+  onAccept
+}) => {
+  return (
+    <Box sx={styles.root}>
+      <Box>
+        <Box sx={styles.title}>
+          <ErrorOutlineRounded />
+          <Typography>Cooperation closing process</Typography>
```

Comment:
it might be better to use text from useTranslation hook instead of hardcoding it ( and everywhere you don't use useTranslation hook )